packages/transformer/src/IModelImporter.ts

```diff
@@ -110,17 +103,17 @@ export class IModelImporter {
 
   /**
    * A set of elementIds that the transformer adds to while exporting elements to indicate that the element already exists in the target.
-   * Defaults to an empty set.
+   * Defaults to undefined.
    * @note
    *
-   * This is used as an optimization when `preserveElementIdsForFiltering` is set to `true`
+   * This is used as an optimization when the transformer `preserveElementIdsForFiltering` option is set to `true`
```

Comment:
maybe we should reference imodeltransformoptions here?

Reply:
Added